submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/service/SubmissionRepository.kt

```diff
@@ -34,6 +34,7 @@ class SubmissionRepository(
         submissionRepository.findByTypeAndAccNo(Project.value, tags.map { it.name })
             .map { submissionDbMapper.toSubmission(it) }
 
+    // TODO this method should return the submission model, not the persistence object
```

Comment:
do not get this TODO, this one is a spring data repository

Reply:
In all the other methods, we return the submission model object but in this case, we return the persistence object. This method should also use the Submission Mapper and return a Submission model object

Reply:
ohh got it I was thinking this was `SubmissionDataRepository` and not `SubmissionRepository`